plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/shell/AbstractShell.java

```diff
@@ -97,13 +96,9 @@ public boolean isFocused() {
 	public void close() {
 		String text = getText();
 		log.info("Close shell " + text);
-		try {
-			new CancelButton().click();
-		} catch (Exception e) {
-			WidgetHandler.getInstance().notify(SWT.Close, swtShell);
-			ShellHandler.getInstance().closeShell(swtShell);
-		}
-		new WaitWhile(new ShellWithTextIsAvailable(text));
+		//WidgetHandler.getInstance().notify(SWT.Close, swtShell);
```

Comment:
Reasons for update:
- we should not even use Cancel button here, sice method is called close, not cancel
- calling notify and then close is the same thing if you take a look at implementation of close method
- its better/safer to use ShellIsAvailable than ShellWithTextIsAvailable since we already have a shell, not just a text